cmd/oras/push.go

```diff
@@ -132,7 +138,15 @@ func runPush(opts pushOptions) error {
 
 	// ready to push
 	resolver := newResolver(opts.username, opts.password, opts.configs...)
-	return oras.Push(context.Background(), resolver, opts.targetRef, store, files, pushOpts...)
+	desc, err := oras.Push(context.Background(), resolver, opts.targetRef, store, files, pushOpts...)
+	if err != nil {
+		return err
+	}
+
+	if opts.verbose {
+		fmt.Printf("pushed %s: %s\n", opts.targetRef, desc.Digest)
```

Comment:
NIT: Similar to the previous comment. Capitalize the start. 
Also is push message in verbose while pull message isn't?

Reply:
Both are in verbose